### ISSUE TITLE
Untranslatable app

### ISSUE THREAD

**Issue by kugiigi** [kugiigi/talaan-app#1]
## Description
Please Add the translations on the launchpad.

Update: Seems to be easier to do in Github?

### Original Link:
https://bugs.launchpad.net/talaan-app/+bug/1565600

**Comment by kugiigi** [comment]
Not sure how that site works and if my app qualifies for it and unfortunately, I don't have much time to tinker with it at the moment. So you tried opening my po files with poedit and it fails?
There are already Spanish and Dutsch translation and maybe you can check them.
Thanks for the interest 😄

**Comment by Danfro** [comment]
I have to a german .po file now.
I did build a .click to test.
There are still a couple of untranslatable strings, framework is stated as 15.04 and version is given as 2.30 at github.
Is the github code up to date? Because there is 2.35 available at open store and running under 16.04.

**Comment by kugiigi** [comment]
> I have to a german .po file now.
> I did build a .click to test.
> There are still a couple of untranslatable strings, framework is stated as 15.04 and version is given as 2.30 at github.
> Is the github code up to date? Because there is 2.35 available at open store and running under 16.04.

This is my fault, I created the xenial version (2.35) like a quick hack and published it directly to the store. And after that I forgot about it and wasn't able to commit in github. I'm currently looking for the files I used for the xenial version and upload them here. And I guess I can close this issue afterwards since there are already translation available. Any other translation issues can be logged separately. Thank you very much for your interest on this app
